README.md

```diff
@@ -644,6 +644,43 @@ Status: 201 created
 }
 
 ```
+#### Create a contact with a company name from the dropdown box
+Request:
+```
+POST api/v1/users/2/companies/2/contacts
+
+Authorization: Bearer Token - put in token for user
+
+raw json body with all fields: 
```

Comment:
Perhaps adding some clarification on which of the fields are absolutely required. I saw in the tests you were using a minimal params when creating. So while it is very helpful to have the full JSON as an example, having some information on which fields will trigger the 404 error if not included can clear that up. For example, first and last name are required while email is an optional field.

Reply:
Thank you for pointing this out. Due to how the DB is currently set up only the first name, last name, and company email, depending on which URL being used. I will update the README right now to make note of when these fields are required.

Reply:
The Read Me has been updated.